### PR TITLE
docs: mention the types without types feature

### PR DESCRIPTION
**Pull request by dummdidumm**
### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [ ] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. Changesets that add features should be `minor` and those that fix bugs should be `patch`. Please prefix changeset messages with `feat:`, `fix:`, or `chore:`.

### CODE REVIEW

**Comment by benmccann** [comment]
I wonder if we should change the other docs and `create-svelte` to remove the usage of `$types` and then in the IDE section say you'll need to use `$types` if you want to use something other than VS Code?

**Comment by dummdidumm** [comment]
I'm against this. In the docs these types provide valuable context to me. And since not all IDEs - intellij in particular - can't take advantage of it, it would be confusing for those who can't use it

File: documentation/docs/20-core-concepts/10-routing.md [hunk 1]
```diff
@@ -348,6 +348,10 @@ For example, annotating `export let data` with `PageData` (or `LayoutData`, for
 
 In turn, annotating the `load` function with `PageLoad`, `PageServerLoad`, `LayoutLoad` or `LayoutServerLoad` (for `+page.js`, `+page.server.js`, `+layout.js` and `+layout.server.js` respectively) ensures that `params` and the return value are correctly typed.
 
+If you're using VS Code or any IDE that supports the language server protocol and TypeScript plugins then you can omit these types _entirely_! Svelte's IDE tooling will insert the correct types for you, so you'll get type checking without writing them yourself. It also works with our command line tool `svelte-check`.
+
+You can read more about `$types` in our [blog post](https://svelte.dev/blog/zero-config-type-safety) about it.
```
Comment (benmccann) [inline documentation/docs/20-core-concepts/10-routing.md:353]:
```suggestion
You can read more about omitting `$types` in our [blog post](https://svelte.dev/blog/zero-config-type-safety) about it.
```

**Comment by benmccann** [comment]
Yeah, you have me convinced. Referencing the type is helpful just for understanding the type of things when reading the docs

The other thing I wonder is if this should go under "Editor setup" rather than the routing section

**Comment by dummdidumm** [comment]
Why_not_both.gif

**Review by benmccann — APPROVED** [review]

**Pull request merged**